pkg/registry/authorization/util/helpers.go

```diff
@@ -77,3 +77,13 @@ func AuthorizationAttributesFrom(spec authorizationapi.SubjectAccessReviewSpec)
 
 	return authorizationAttributes
 }
+
+const AllVersion = "*"
```

Comment:
no need to export a constant from here, inline the `*` in `matchAllVersionIfEmpty` function